hw/top_earlgrey/cdc/cdc_waivers.w_interface.tcl

```diff
@@ -19,7 +19,7 @@ set_rule_status -rule {W_INTERFACE} -expression {(Signal =~ "top_earlgrey.u_spi_
 set_rule_status -rule {W_INTERFACE} -expression {(Signal =~ "top_earlgrey.u_spi_device.u_reg.u_rxf_ptr_rptr.q[12:2]")} -status {Waived} -comment {included in waived paths : start signal and receiving signal (flop) have been reviewed and waived in the same error or other errors}
 set_rule_status -rule {W_INTERFACE} -expression {(Signal =~ "top_earlgrey.u_spi_device.u_fwmode.u_rx_fifo.storage*")} -status {Waived} -comment {included in waived paths : start signal and receiving signal (flop) have been reviewed and waived in the same error or other errors}
 set_rule_status -rule {W_INTERFACE} -expression {(Signal =~ "top_earlgrey.u_spi_device.u_fwmode.u_rx_fifo.fifo_wptr_gray_q[3:0]")} -status {Waived} -comment {included in waived paths : start signal and receiving signal (flop) have been reviewed and waived in the same error or other errors}
-set_rule_status -rule {W_INTERFACE} -expression {(Signal =~ "top_earlgrey.u_spi_device.io_mode_outclk[1:0]"} -status {Waived} -comment {included in waived paths : start signal and receiving signal (flop) have been reviewed and waived in the same error or other errors; set_rule_status run by  user root on host smart-dis-us-central1-ac9805fe-454c-4151-83d0-2daae3aac116 at Tuesday, 15 November 2022 14:56:22 PDT}
+set_rule_status -rule {W_INTERFACE} -expression {(Signal =~ "top_earlgrey.u_spi_device.io_mode_outclk[1:0]")} -status {Waived} -comment {included in waived paths : start signal and receiving signal (flop) have been reviewed and waived in the same error or other errors; set_rule_status run by  user root on host smart-dis-us-central1-ac9805fe-454c-4151-83d0-2daae3aac116 at Tuesday, 15 November 2022 14:56:22 PDT}
```

Comment:
can you remove the smart ID here? in general, it would be good to do a search through all waiver files and remove this information.

Reply:
I removed them. As I searched, no more smart ID is found after this fix.